### DISCUSSION TITLE
MongoDB Data Proxy

### DISCUSSION THREAD

**Discussion by willin** [prisma/prisma#11847] [Q&A] [answered]
```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["mongoDb", "dataProxy"]
}
```

i see several examples using so, but i don't found any docs about data-proxy of mongodb.

**Comment by austincrim** [comment, marked as answer]
Hey @willin 👋 ,

The Data Proxy should function the same regardless of database provider. You can read more about how to use the Data Proxy [in our docs](https://www.prisma.io/docs/concepts/data-platform/data-proxy#using-the-data-proxy-in-a-prisma-application).

Let me know if I can help with anything!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
